Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is the second line.
```

Comment:
@abirendra look at this!

---
In reply to: [61154868](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r61154868) [](ancestors = 61154868,59789954,59789925)